src/lib/md.mli

```diff
@@ -48,4 +48,6 @@ val process_publish_block :
   Cmarkit.Block.Code_block.t * Block.t
 
 val translate_import_block :
-  uid:string -> Block.t -> Cmarkit.Block.Code_block.t * Block.t
+  uid:string ->
+  Block.t ->
+  (Cmarkit.Block.Code_block.t * Block.t) * string option
```

Comment:
Can we add a comment about what that new, extra `string option` argument is -- maybe we keep it as an `Fpath.t` until the caller needs to translate it ?